themes/jquery/js/modal.js

```diff
@@ -0,0 +1,53 @@
+// Store modal templates
```

Comment:
Rename this file to `sri.js` or `sri-modal.js`.

Wrap the entire contents in an IIFE so you're not creating global variables.